katharsis-core/src/main/java/io/katharsis/queryspec/repository/QuerySpecResourceRepository.java

```diff
@@ -15,6 +15,7 @@
  * QuerySpec-based ResourceRepository that translates incoming QueryParameters
  * to QuerySpec for ease of use.
  */
+@Deprecated
 public abstract class QuerySpecResourceRepository<T, I extends Serializable>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1133)